org.eclipse.wildwebdeveloper.tests/src/org/eclipse/wildwebdeveloper/tests/TestLanguageServers.java

```diff
@@ -233,5 +248,70 @@ protected boolean condition() {
 			}
 		}.waitForCondition(PlatformUI.getWorkbench().getDisplay(), 3000));
 	}
+	
+	@Test
+	public void testAngularTSFile() throws Exception {
+		URL url = FileLocator.find(Platform.getBundle("org.eclipse.wildwebdeveloper.tests"), Path.fromPortableString("testProjects/angular-app"), null);
+		url = FileLocator.toFileURL(url);
+		File folder = new File(url.getFile());
+		if (folder.exists()) {
+			FileUtils.copyDirectory(folder, project.getLocation().toFile());
+			Process process = new ProcessBuilder(getNpmLocation(), "install", "--no-bin-links", "--ignore-scripts").directory(project.getLocation().toFile()).start();
+			while (process.isAlive());
+			project.refreshLocal(IResource.DEPTH_INFINITE, new NullProgressMonitor());			
+			IFile file = project.getFolder("src").getFolder("app").getFile("app.component.ts");
+			String fileContent = IOUtils.toString(file.getContents(), StandardCharsets.UTF_8);
+			ITextEditor editor = (ITextEditor) IDE.openEditor(PlatformUI.getWorkbench().getActiveWorkbenchWindow().getActivePage(), file);
+			editor.getDocumentProvider().getDocument(editor.getEditorInput()).set(fileContent);
+			assertTrue("Diagnostic not published", new DisplayHelper() {
+				@Override
+				protected boolean condition() {
+					try {
+						return file.findMarkers("org.eclipse.lsp4e.diagnostic", true, IResource.DEPTH_ZERO).length != 0;
+					} catch (CoreException e) {
+						return false;
+					}
+				}
+			}.waitForCondition(PlatformUI.getWorkbench().getDisplay(), 50000));
+			
+			IMarker [] markers = file.findMarkers("org.eclipse.lsp4e.diagnostic", true, IResource.DEPTH_ZERO);
+			boolean foundError = false;
+			for (IMarker marker : markers) {
+				int lineNumber = marker.getAttribute(IMarker.LINE_NUMBER, -1);
+				if (lineNumber == 6) {
+					foundError = true;
```

Comment:
Would be good to check also for a keyword like "Component" in the marker message to ensure the message comes from the right LS.